### PR TITLE
update aws doc to remove secrets, focus on roles

### PR DESCRIPTION
**Pull request by technovangelist**

### CODE REVIEW

**Comment by tmichelet** [comment]
looks good, could we see that live on staging?

**Comment by technovangelist** [comment]
thanks @tmichelet . just did a merge into staging

File: content/integrations/aws.md [hunk 1]
```diff
@@ -182,21 +154,21 @@ written a bit about this [here][7],  especially in relation to CloudWatch.
...
+**Wrong count of aws.elb.healthy_host_count?**
 
 When the Cross-Zone Load Balancing option is enabled on an ELB, all the instances attached to this ELB are considered part of all A-Zs (on cloudwatch’s side), so if you have 2 instances in 1a and 3 in ab, the metric will display 5 instances per A-Z.
 As this can be counter-intuitive, we’ve added a new metric, aws.elb.host_count, that displays the count of healthy instances per AZ, regardless of if this Cross-Zone Load Balancing option is enabled or not.
 This metric should have value you would expect.
 
 
 
-## Duplicated hosts when installing the agent?
+**Duplicated hosts when installing the agent?**
```
Comment (tmichelet) [inline content/integrations/aws.md]: @technovangelist why do we want to update that? It's very convenient to be able to reference to those sections in one click :)

**Comment by technovangelist** [comment]
we want to create headings for the sections, but not for individual questions. this becomes consistent with the rest of the docs.

**Comment by tmichelet** [comment]
Could there be a way to add anchors to individual questions? I use this feature on 50% of aws related support cases

**Comment by technovangelist** [comment]
ok, @tmichelet your request is done
 
is the feature live? can we merge?

File: content/integrations/aws.md [hunk 2]
```diff
@@ -67,9 +55,9 @@ There are a number of other AWS services that are also available in Datadog but
 
 # Installation
 
-There are two integration methods that can be used to allow Datadog to monitor your AWS environment. Both require creating a policy in the AWS Console with a certain set of permissions. The difference between the two is whether you choose to create a role that Datadog has access to which is preferred due to the higher level of security, or create a user and share an AWS Secret and Access Key. To get a better understanding of role delegation, refer to the [AWS IAM Best Practices guide](http://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#delegate-using-roles).
+Setting up the Datadog integration with Amazon Web Services requires configuring role delegation using AWS IAM. To get a better understanding of role delegation, refer to the [AWS IAM Best Practices guide](http://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#delegate-using-roles).
 
-Note: GovCloud does not support role based authentication.
+Note: If you are using GovCloud or have resources on AWS China, contact support to help set up your integration.
```
Comment (irabinovitch) [inline content/integrations/aws.md]: Note: The GovCloud and China reasons do not currently support IAM role delegation.  If you are deploying in these reasons please contact support to help set up your AWS integration via API keys.

**Comment by irabinovitch** [comment]
@technovangelist  yes I can confirm that production no longer shows the option to specify keys as credentials.

**Comment by tmichelet** [comment]
thank you @technovangelist

**Pull request merged**